tasks/install_agent_linux.rb

```diff
@@ -0,0 +1,61 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
This file is pretty big to not have any docs. Please add some docs to describe what it is doing and perhaps some inline ones as you see fit.

Reply:
I added some inline but the main documentation is handled by Puppet Strings.